### PR TITLE
DDF-3880 Resolve mvn build warnings

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do?
Fixes build issues printed out at the top of build concerning misconfigured pom files
(like duplicate dependencies, missing versions, etc).

#### Who is reviewing it? 
anyone
#### How should this be tested? (List steps with links to updated documentation)
build
#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-3880
#### Screenshots (if appropriate)
**FROM THIS:**
screenshot 2018-06-07 16 54 46

**TO THIS:**
screenshot 2018-06-07 17 06 10

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by rzwiefel** [comment]
build now

File: catalog/ui/catalog-ui-search/pom.xml [hunk 1]
```diff
@@ -297,6 +297,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.0.0</version>
```
Comment (coyotesqrl) [inline catalog/ui/catalog-ui-search/pom.xml]: :question: Why not the variable here?

Comment (rzwiefel) [reply catalog/ui/catalog-ui-search/pom.xml]: Because I forgot to add it here

Comment (peterhuffer) [inline catalog/ui/catalog-ui-search/pom.xml]: ❓ Can we use `maven-checkstyle-plugin.version` property here?

Comment (rzwiefel) [reply catalog/ui/catalog-ui-search/pom.xml]: Fixed

**Comment by rzwiefel** [comment]
Aborted previous job, pushed new changes, Build now

**Pull request merged**